### PR TITLE
Device update code sample

### PR DESCRIPTION
**Pull request by razor-x**
- Note unmanaging codes when unmanaging a device
- Add note about unmanaging access codes
- Add device update code sample

### CODE REVIEW

File: docs/api-clients/devices/list-unmanaged-devices.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ description: Get all Unmanaged Devices
 ---
 
 An unmanaged device has a limited set of visible properties, a subset of supported events, and may not be controlled. 
+Any access codes on an unmanaged device will be unmanaged.
```
Comment (rchodava) [inline docs/api-clients/devices/list-unmanaged-devices.md:6]: Maybe:
All access codes on an unmanaged device are also unmanaged

File: docs/api-clients/devices/update-device.md [hunk 2]
```diff
@@ -45,3 +50,19 @@ Name of Property
...
+{% tabs %}
+{% tab title="Python" %}
+```python
+seam.devices.update(properties={"name": "My Lock"})
+```
+{% endtab %}
+
+{% tab title="Javascript" %}
+```typescript
+await seam.devices.unmanaged.update({ properties: { name: "My Lock" } })
```
Comment (rchodava) [inline docs/api-clients/devices/update-device.md:65]: hmm should there be a `is_managed: false` flag you pass in to the update method?

**Review by rchodava — APPROVED** [review]

File: docs/api-clients/devices/list-unmanaged-devices.md (hunk 1 above)
Comment (razor-x) [reply docs/api-clients/devices/list-unmanaged-devices.md:6]: I had something like this at first, but I prefer the stronger / active voice. I have a personal vendetta against the word `also`, but that's a different story 😅

File: docs/api-clients/devices/update-device.md (hunk 2 above)
Comment (razor-x) [reply docs/api-clients/devices/update-device.md:65]: Well, this was just missing docs for updating a device. I didn't want to default example to unmanage the device. Not a great UX for people playing with the API IMO.

**Pull request merged**